src/en/about/dat/terms-of-reference.md

```diff
@@ -0,0 +1,137 @@
+- improve consistency of accessibility and identify best practices for Information and Communication Technology (<abbr>ICT</abbr>) accessibility information;
+- increase awareness across participating departments through the Access Working Group (<abbr>AWG</abbr>) and Interdepartmental <abbr title="Information and Communication Technology">ICT</abbr> working Group;
+- leverage expertise, foster information flow, and broaden knowledge across the <abbr title="Government of Canada">GC</abbr>; and
+- make recommendations for future phases of the project -- for example expanding access to other jurisdictions that are collaborating with the <abbr title="Government of Canada">GC</abbr> on <abbr title="Information and Communication Technology">ICT</abbr> accessibility.
+
+## Membership
+
+Digital Accessibility Toolkit (<abbr>DAT</abbr>) members belong to multiple stakeholder departments who have knowledge in digital accessibility.
+
+Members
+
+- Co-Chair(s):
+  - Shared Service Canada (<abbr>SSC</abbr>): Shawn Thompson, Accessibility, Accommodation and Adaptive Computer Technology (<abbr>AAACT</abbr>) Program (Development lead)
+  - Canada School of Public Service (<abbr>CSPS</abbr>): Julie Bélisle Boutet, Digital Accessibility in Learning (Content lead)
+- Contributing resources:
+  - Open Government Working Group
+  - Canadian Digital Services (<abbr>CDS</abbr>)
+- Different subject matter experts, other than accessibility
+  - Legal
+  - Plain language
+  - Official languages
+  - Open data
+  - Development
+  - Communications
+- Secretariat services:
+  - Accessibility, Accommodation and Adaptive Computer Technology (<abbr>AAACT</abbr>) program
+
+## Operating procedures and frequency of meetings
+
+This task force meets weekly, or as required. Agendas will be prepared for each meeting and meeting materials will be circulated in advance of the meetings. Members are invited at all times to add agenda items.
+
+Members are expected to be working members, drafting and editing materials and making recommendations.
+
+In the event that the member is unavailable for a scheduled meeting, the member may appoint an alternate who has equivalent background and/or knowledge of the subject being discussed.
+
+A brief summary of the project progress will be provided at minimum at every other Access Working Group (<abbr>AWG</abbr>) meeting.
+
+The methods of contributing could be as done in one or more of the following ways:
+
+1. [Github discussion forum](https://github.com/gc-da11yn/gc-da11yn.github.io/discussions) in the open;
+    1. GitHub,
+    2. email, or
+    3. anonymously via web form.
+2. Task force meetings
+
+Final decisions will be published on the [Github discussion forum](https://github.com/gc-da11yn/gc-da11yn.github.io/discussions).
+
+The Terms of Reference will be reviewed at minimum annually and adjusted as required.
+
+## Scope
+
+While ideally the project have a four phases approach, the focus of this task force will be on Phase 1 only.
```

Comment:
Be a little more time specific. We might forget to update that time reference when we will be at future phase.

```suggestion
While ideally the project have a four phases approach, the focus of this task force will be on Phase 1 only as early 2023.
```